app/modules/individuals/parameters.py

```diff
@@ -53,16 +61,113 @@ def remove(cls, obj, field, value, state):
         elif field == 'featuredAssetGuid' and util.is_valid_guid(value):
             obj.set_featured_asset_guid(UUID(value, version=4))
             ret_val = True
+        elif field == 'names' and isinstance(value, str) and util.is_valid_guid(value):
```

Comment:
Surely a safer change would be to change utils_is_valiid_guid to check that  value was a str. 
The check seems essential in all cases so no need to have it just here

Reply:
done!